### ISSUE TITLE
If server fails it refuses to restart

### ISSUE THREAD

**Issue by mitra42** [mitra42/bct-server#99]
If I run the server with
```
python3 ./server.py --config_file sample_global_config.ini
```
and then ctrl-c out of it then a subsequent run fails until I do 
```
rm -r /tmp/spatial_dict /tmp/contact_dict
```
with 
```
`2020-05-03 19:00:03+1000 [-]   File "/Users/mitra/git/bct-server/contacts.py", line 73, in _load
2020-05-03 19:00:03+1000 [-]     self._load_key(code, blob)
2020-05-03 19:00:03+1000 [-]   File "/Users/mitra/git/bct-server/contacts.py", line 269, in _load_key
2020-05-03 19:00:03+1000 [-]     self.keys[key_tuple] = key_string
2020-05-03 19:00:03+1000 [-] AttributeError: 'SpatialDict' object has no attribute 'keys'
```
I'm not sure what is happening ...

**Comment by danaronson** [comment]
i can not duplicate.  Can you give me a list of things that you did after you started the server?

**Comment by danaronson** [comment]
fixed in #101
